# HG changeset patch
# User Rik <rik@octave.org>
# Date 1494529364 25200
#      Thu May 11 12:02:44 2017 -0700
# Branch stable
# Node ID 2f90ac754550a588ef430e816b28d4a98ee65d5e
# Parent  0ada77ed24dda7aef2b531c1a446bb98b6bce8b6
print.m: Update documentation to refer to OpenGL-based toolkits, not just FLTK.

* print.m: Update documentation to refer to OpenGL-based toolkits, not just FLTK.

diff --git a/scripts/plot/util/print.m b/scripts/plot/util/print.m
--- a/scripts/plot/util/print.m
+++ b/scripts/plot/util/print.m
@@ -86,25 +86,25 @@
 ##   The available output format is specified by the option @var{device}, and
 ## is one of:
 ##
 ##   @table @code
 ##   @item  ps
 ##   @itemx ps2
 ##   @itemx psc
 ##   @itemx psc2
-##     PostScript (level 1 and 2, mono and color).  The FLTK graphics
-## toolkit generates PostScript level 3.0.
+##     PostScript (level 1 and 2, mono and color).  The OpenGL-based toolkits
+## always generate PostScript level 3.0.
 ##
 ##   @item  eps
 ##   @itemx eps2
 ##   @itemx epsc
 ##   @itemx epsc2
-##     Encapsulated PostScript (level 1 and 2, mono and color).  The FLTK
-## graphic toolkit generates PostScript level 3.0.
+##     Encapsulated PostScript (level 1 and 2, mono and color).  The
+## OpenGL-based toolkits always generate PostScript level 3.0.
 ##
 ##   @item  pslatex
 ##   @itemx epslatex
 ##   @itemx pdflatex
 ##   @itemx pslatexstandalone
 ##   @itemx epslatexstandalone
 ##   @itemx pdflatexstandalone
 ##     Generate a @LaTeX{} file @file{@var{filename}.tex} for the text
@@ -191,20 +191,21 @@
 ##     Portable document format
 ##   @end table
 ##
 ##   @item svg
 ##     Scalable vector graphics
 ##
 ##   @item  tikz
 ##   @itemx tikzstandalone
-##     Generate a @LaTeX{} file using PGF/TikZ@.  For the FLTK toolkit
-## the result is PGF@.  The @samp{tikzstandalone} device produces a @LaTeX{}
-## document which includes the TikZ file (@samp{tikzstandalone} is only
-## available for the Gnuplot graphics toolkit).
+##     Generate a @LaTeX{} file using PGF/TikZ format.  The OpenGL-based
+## toolkits create a PGF file while Gnuplot creates a TikZ file.  The
+## @samp{tikzstandalone} device produces a @LaTeX{} document which includes the
+## TikZ file (@samp{tikzstandalone} and is only available for the Gnuplot
+## graphics toolkit).
 ##
 ##   If the device is omitted, it is inferred from the file extension,
 ## or if there is no filename it is sent to the printer as PostScript.
 ##
 ## @item -d@var{ghostscript_device}
 ##   Additional devices are supported by Ghostscript.
 ## Some examples are;
 ##
